src/atom.coffee

```diff
@@ -783,11 +783,12 @@ class Atom extends Model
   showSaveDialog: (callback) ->
     callback(showSaveDialogSync())
 
-  showSaveDialogSync: (defaultPath) ->
-    defaultPath ?= @project?.getPaths()[0]
+  showSaveDialogSync: (options={}) ->
```

Comment:
I know this isn't a public API but I think it might be nice to still preserve the old behavior.

Basically just test of the first argument is a string and retain the old behavior in this case.

Does that sound okay?